cmd/chirpstack-application-server/cmd/configfile.go

```diff
@@ -207,6 +207,20 @@ id="{{ .ApplicationServer.ID }}"
     # The login label is used in the web-interface login form.
     login_label="{{ .ApplicationServer.UserAuthentication.OpenIDConnect.LoginLabel }}"
 
+    # Use UserInfo.
+    #
+    # Use a UserInfo call to retrieve the claim rather than extracting the claim from
+    # the idToken.
+    use_userinfo="{{ .ApplicationServer.UserAuthentication.OpenIDConnect.UseUserInfo }}"
```

Comment:
Could you set the default to `true`? Else an update will very likely be breaking for existing setups. Please see:
https://github.com/brocaar/chirpstack-application-server/blob/master/cmd/chirpstack-application-server/cmd/root.go#L31.

Alternatively, reverse the config option. E.g. instead of enabling the use of the `UserInfo endpoint`, it would disable it.

Reply:
Thanks for the reference. I've added defaults there for both of these options.